Jenkinsfile.aws-test

```diff
@@ -71,6 +84,12 @@ node(NODE) {
                     aws s3 cp --acl public-read \
                         ${WORKSPACE}/aws-${AWS_REGION}.json \
                         s3://${S3_PUBLIC_BUCKET}/aws-${AWS_REGION}-tested.json
+
+                    # Tag the container image to alpha, then GC the image tagged with the ostree commit
+                    podman pull ${OSCONTAINER_IMG}:${ostree_commit}
+                    podman tag ${OSCONTAINER_IMG}:${ostree_commit} ${OSCONTAINER_IMG}:alpha
+                    podman push ${OSCONTAINER_IMG}:alpha
```

Comment:
We don't actually need to pull locally here, we can just `skopeo copy` right?

Reply:
Ooh, fancy!